AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs

```diff
@@ -411,5 +411,29 @@ private static void RetrieveFirstAndLastNameFromExternalPrincipal(ExternalLoginI
             firstName = array[0];
             lastName = array[1];
         }
+        
+        //Area Names
+        private const string ADMIN_AREA = "Admin";
+
+        //Controller Names
+        private const string HOME_CONTROLLER = "Home";

```

Comment:
nameof(HomeController) results in "HomeController". Do you want me to add some string operation, like nameof(HomeController).Replace("Controller","")? Looks a bit messy, imho. 

Re nameof area, I can't seem to find that implemented. nameof() documentation says it is "used to obtain the simple string name of a variable, type, or member". Not sure any of those can be applied to area, it is defined as a "logical grouping". Please correct me if i'm wrong.